Power Up Proto/src/org/usfirst/frc/team5493/robot/Robot.java

```diff
@@ -23,7 +23,7 @@
 	public static final DriveBase driveBase = new DriveBase();
 	public static OneClimbyBoi climber;
 	public static OI oi;
-	public static ThrowDaggersInBensEyes throwDaggersInBensEyes;
+//	public static ThrowDaggersInBensEyes throwDaggersInBensEyes;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)